### PR TITLE
Switch from webpack to vite

### PR DESCRIPTION
**Pull request by LudvigHz**
Yeet webpack.
Yank vite.js

This removes a bunch of packages needed for webpack, simplifies the config _alot_, speeds up the build and dev server, and makes the bundling/packaging better and smoother.

### CODE REVIEW

**Review by SmithPeder — APPROVED** [review]
Nice 😍

**Review by eikhr — CHANGES_REQUESTED** [review]
Looks great! 

But rename `vite.config.js` to `vite.config.ts`, that will make it give type errors if the config has invalid values. Seems like `rollupOptions` is supposed to be within `build`, and it wants the port to be a number.

**Review by eikhr — APPROVED** [review]
Perfect!

File: .drone.yml [hunk 1]
```diff
@@ -17,7 +17,14 @@ steps:
       event: push
     commands:
       - yarn lint
-      -
+
+  - name: Type check
```
Comment (ivarnakken) [inline .drone.yml:21]: btw, I hate that you capitalized the first letter here, we sort of have a schema going on y'know 😣

**Pull request merged**